google-brokers/manifest.yml

```diff
@@ -0,0 +1,19 @@
+packversion: 1
+name: google-services
+version: 1.0.0
+metadata:
+  author: jlewisiii@google.com
```

Comment:
I would shy away from using my real email address in example tests, but your call :)